motr/st/utils/helper.c

```diff
@@ -250,6 +254,12 @@ static int create_object(struct m0_entity *entity)
 			       M0_BITS(M0_OS_FAILED,
 				       M0_OS_STABLE),
 				       M0_TIME_NEVER);
+	oc = M0_AMB(oc, *ops, oc_op);
```

Comment:
s
/oc = M0_AMB(oc, *ops, oc_op);
/oc = M0_AMB(oc, ops[0], oc_op);

This means the first m0_op pointer in this array.